src/spikeinterface/sorters/runsorter.py

```diff
@@ -414,9 +415,15 @@ def run_sorter_container(
 
     # create 3 files for communication with container
     # recording dict inside
-    (parent_folder / "in_container_recording.json").write_text(
-        json.dumps(check_json(rec_dict), indent=4), encoding="utf8"
-    )
+    if recording.check_serializability("json"):
```

Comment:
Not that it matters but there is a reason you are not using `recording.dump()` directly?

Reply:
@h-mayorquin also want to change this but not in this PR because it imply some path handling and relative_to refactroing.

Reply:
Yes, that makes sense. I was not suggesting that we should do that here. I was asking questions to my benefit and understanding.

This is what I want to convey when I approve a PR but then add comments: "This is good and I think is fine as it is  but what about this or that now that we are on the topic"

In this case it was mere curiosity. Thanks for engaging with me guys.